drivers/ds323x/include/ds3234_params.h

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Do you think it is necessary to have separate *params* files?

Reply:
Since ds3234 is spi connected and the others are i2c, I figured the configuration should be separate